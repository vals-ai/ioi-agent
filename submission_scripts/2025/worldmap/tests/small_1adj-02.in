version https://git-lfs.github.com/spec/v1
oid sha256:06c00fd649d8d335c76703b480ea4db69c5a9c93a1ea5962c2795c0ad62f45d3
size 2051
