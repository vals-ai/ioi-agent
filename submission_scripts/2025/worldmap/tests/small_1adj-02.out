version https://git-lfs.github.com/spec/v1
oid sha256:7989da7893ebd72ee01827b459ca7446d1a934bbda411df7e7865e11b2da8b64
size 22495
